pkg/cmd/server/start/start_master.go

```diff
@@ -581,7 +592,9 @@ func startControllers(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) erro
 
 		// called by admission control
 		kc.RunResourceQuotaManager()
-		oc.RunResourceQuotaManager(kc.ControllerManager)
+		if runOriginResourceQuotaController {
+			oc.RunResourceQuotaManager(kc.ControllerManager)
```

Comment:
Yes, currently it means to disable handling of just `openshift.io/imagestreamimages` and `openshift.io/imagestreamtags`. This construct will be removed after 3.2 is out though.